src/services/Game.js

```diff
@@ -73,6 +77,22 @@ export default class Game {
     return this.#lap;
   }
 
+  /**
+   * 우승한 자동차 이름들을 추출하는 함수
+   *
+   * @param {Car[]} cars 마지막 바퀴까지 달린 자동차들
+   * @returns 우승한 자동차 이름들
+   */
+  getWinners(cars) {
+    const carLocations = cars.map((car) => car.getLocation());
+    const topTrack = Math.max(...carLocations);
+
+    return cars
+      .filter((car) => topTrack === car.getLocation())
+      .map((car) => car.getName())
+      .join(', ');
+  }
```

Comment:
우승한 자동차들의 "이름"을 반환하고 있다는 것이 함수명을 통해 더 잘 드러났으면 좋겠네용
```suggestion
  getWinnersName(cars) {
    const carLocations = cars.map((car) => car.getLocation());
    const topTrack = Math.max(...carLocations);

    return cars
      .filter((car) => topTrack === car.getLocation())
      .map((car) => car.getName())
      .join(', ');
  }
```